constellation-app/constellation
# Fix github testing issues

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change


Fixed test files that would fail randomly and inconsistently. The two main causes were a concurrency problem in QueryPhasePane and its tests, and similar issues in ValueInputPane when binding a property in a run later.

### Alternate Designs


N/A
### Why Should This Be In Core?


N/A
### Benefits


Unit tests are far less likely to fail

### Possible Drawbacks



### Verification Process


To validate the ValueInputPane still functions as intended:

1. Open data access view
2. Check "Test Parameters"
3. Click the down drop (recent values list) next to"Test1" parameter and observe no recent values
4. Type a value into "Test1" parameter
5. Click "GO"
6. Click the down drop (recent values list) next to"Test1" parameter
7. Observe the previous string entered is in the down drop 

Can only really verify tests by re-running the tests on this PR and observing they don't fail. In my experience the tests were more likely to randomly fail on GitHub, rather than locally.

### Applicable Issues


N/A
